wdtk-client/src/main/java/org/wikidata/wdtk/client/ClientConfiguration.java

```diff
@@ -142,6 +146,11 @@
 	 */
 	public static final String OPTION_FILTER_PROPERTIES = "fProp";
 	/**
+	 * Name of the option to print a report about the files produced by
+	 * DumpProcessingOutputActions
```

Comment:
Misleading documentation. The option is not "to print a report" but to define the output file to print the report to. The report will only be "printed" if the option is not set, right?